File: backend/databasehandler/src/main/java/com/pixtranslator/backend/databasehandler/DatabasehandlerApplication.java
package com.pixtranslator.backend.databasehandler;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "com.pixtranslator.backend")
public class DatabasehandlerApplication {

  public static void main(String[] args) {
    SpringApplication.run(DatabasehandlerApplication.class, args);
  }

}

File: backend/databasehandler/src/main/kotlin/com/pixtranslator/backend/databasehandler/DatabasehandlerApplication.kt
package com.pixtranslator.backend.databasehandler

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackages = ["com.pixtranslator.backend"])
open class DatabasehandlerApplication {
  fun main() {
    SpringApplication.run(DatabasehandlerApplication::class.java)
  }
}